data/transactions/logic/mocktracer/scenarios.go

```diff
@@ -29,6 +30,9 @@ import (
 )
 
 const programTemplate string = `#pragma version 6
+pushbytes "a"
+log
```

Comment:
The changes to this file only add more log statements to app calls to strengthen the tests. 

I probably should have included this in #5071 instead